x/evm/types/utils.go

```diff
@@ -55,14 +55,17 @@ func DecodeTxResponses(in []byte) ([]*MsgEthereumTxResponse, error) {
 	if err := proto.Unmarshal(in, &txMsgData); err != nil {
 		return nil, err
 	}
-
-	responses := make([]*MsgEthereumTxResponse, len(txMsgData.MsgResponses))
-	for i, res := range txMsgData.MsgResponses {
+	responses := make([]*MsgEthereumTxResponse, 0)
```

Comment:
```suggestion
	responses := make([]*MsgEthereumTxResponse, 0, len(txMsgData.MsgResponses))
```

we'll need the capacity most of the time